HW0/P4/P4.py

```diff
@@ -2,6 +2,23 @@
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
 
+def predictS(A, s_k, a):
```

Comment:
I'm not sure if they have to be defined above where the functions are defined but you don't have to pass in global variables as parameters